pom.xml

```diff
@@ -59,7 +59,7 @@
     <jar.manifest>${project.basedir}/src/main/resources/META-INF/MANIFEST.MF</jar.manifest>
     <!-- We skip sources jar generation as we do it with the assembly plugin to have greater control over the content -->
     <source.skip>true</source.skip>
-    <testcontainers.version>1.12.4</testcontainers.version>
```

Comment:
can we stick to the current test container version ?

Reply:
I was not able to start clickhouse testcontainer with 1.12.4. Should not be a problem since it is already upgraded in master: https://github.com/eclipse-vertx/vertx-sql-client/blob/master/pom.xml#L62